### ISSUE TITLE
Agent mode action is confusing for screen reader users

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#239994]
cc @jooyoungseo

1. Focus the agent mode button with the keyboard and a screen reader enabled
2. Hear `Agent Mode Disabled, checkbox unchecked`
3. Toggle it, hear `Agent Mode Enabled, checkbox checked`

We should either keep the label static or do something better here.
![Image](https://github.com/user-attachments/assets/5b43d482-103b-4cea-ae9a-7f2ea8f888e8)

**Comment by roblourens** [comment]
This should be fixed with https://github.com/microsoft/vscode/pull/240014
